geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/iterators/DensityScan.scala

```diff
@@ -183,26 +191,204 @@ object DensityScan {
   /**
     * Writes a density record from a feature that has a point geometry
     */
-  private def writePoint(sf: SimpleFeature, geomIndex: Int, weight: Double, grid: GridSnap, result: DensityResult): Unit =
+  private def writePoint(geomIndex: Int, grid: GridSnap)
```

Comment:
This looks like a method that has two arg groups rather than a method that returns a method.

Reply:
yeah it does have 2 arg groups - it just makes it slightly nicer to partially apply it on lines 155-160